tutorials/getting_started.md

```diff
@@ -0,0 +1,116 @@
+# getting started with napari
+
+Welcome to the getting started with **napari** tutorial!
+
+This tutorial assumes you have already installed **napari**. For help with installation see our [installation tutorial](installation.md).
+
+This tutorial will teach you all the different ways to launch **napari**. At the end of the tutorial you should be able to launch **napari** and see the viewer your favourite way.
+
+
+## launching napari
+
+There are three ways to launch the **napari** viewer:
+- command line
+- python script
+- jupyter notebook / ipython
+
+
+All three of these methods will launch the same **napari** viewer, but depending on your use-case different ones may be preferable.
+
+### command line usage
+
+To launch **napari** from the command line simply run
+```sh
+napari
+```
+
+This command will launch an empty viewer as you can see below.
+
+![image](resources/launch_cli_empty.gif)
+
+Once you have the viewer open you can add images through the `File/Open` dropdown menu or by dragging and dropping images directly on the viewer. We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread), such as `tif`, `png`, and `jpg`. We plan on adding support for more exotic file types shortly - see issue #379 for discussion. You can also create new empty `points`, `shapes`, and `labels` layers using the new layer buttons in the bottom right of the viewer.
+
+You can also directly load an image into the viewer from the command line by passing the path to the image as an argument as follows
+```sh
+napari my_image.png
+```
+If the image is multichannel (i.e. `RGB` or `RGBA`) use the `-m` or `--multichannel` flag.
+
+![image](resources/launch_cli_image.gif)
+
+Launching **napari** directly from the command line is the simplest and fastest way to open the viewer, but it doesn't allow you to preprocess your images before opening them. It is also currently not possible to save images or other layer types directly from the viewer, but we'll be adding support for this functionality soon as discussed in #379.
+
+### python script usage
+
+To launch **napari** from a python script, inside your script you should import `napari`, instantiate a qt gui, and create a `Viewer` object.
```

Comment:
slight preference for "instantiate a Qt GUI"

Reply:
fixed here and elsewhere